keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -459,87 +464,136 @@ Windows privileged containers will be implemented with [Job Objects](https://doc
 
```

Comment:
I can't comment at that line.

> No support for pid and ipc since jobs don’t provide isolation at that level

Just for my education. Does this mean that windows privileged container will see host processes or not?

Reply:
Yes, the can see host processes.

Reply:
I see. Then it seems that we should say "HostIPC" and "HostPID" are always on, because job doesn't have that level of isolation.

The current wording feels a bit confusing when reading.

Reply:
Agreed, will update.